src/dmd/backend/divcoeff.d

```diff
@@ -43,10 +52,10 @@ void u128Div(ullong xh, ullong xl, ullong yh, ullong yl, ullong *pqh, ullong *pq
 
     //ullong xxh = xh, xxl = xl, yyh = yh, yyl = yl;
 
-    assert(yh || yl);           // no div-by-0 bugs
+//    assert(yh || yl);           // no div-by-0 bugs
```

Comment:
Why not use the `dassert` workaround for DAutoTest I proposed here?
https://github.com/dlang/dmd/pull/7714#issuecomment-377394434 
If the body is `version(assert)` it should behave like built-in `assert`? Alternatively it's really just 2.067 that needs special casing.

Or maybe @CyberShadow can update DAutoTest to use 2.068.2 as initial bootstrap compiler?

Reply:
-> https://github.com/CyberShadow/ae/pull/35

Reply:
I don't want to redo the asserts, as that doesn't scale to the rest of the back end. What's needed is to upgrade the boot compiler to 2.074. But that's for after this PR.

Reply:
> as that doesn't scale to the rest of the back end.

Well, I thought it helps porting when the `assert`'s are still enabled, but yeah this can be done in another PR